README.md

```diff
@@ -170,7 +170,7 @@ Following table lists all the parameters supported by the latest MarkLogic Helm
 | `containerSecurityContext.enabled`                  | Parameter to enable security context for MarkLogic containers                                                                                                                          | `true`                     |
 | `containerSecurityContext.runAsUser`                | User ID to run the entrypoint of the container process                                                                                                                                 | `1000`                     |
 | `containerSecurityContext.runAsNonRoot`             | Indicates that the container must run as a non-root user                                                                                                                               | `true`                     |
-| `containerSecurityContext.allowPrivilegeEscalation` | Controls whether a process can gain more privileges than its parent process                                                                                                            | `true`                     |
+| `containerSecurityContext.allowPrivilegeEscalation` | Controls whether a process can gain more privileges than its parent process                                                                                                            | `false`                     |
```

Comment:
readinessProbe.enabled | Parameter to enable the readiness probe | false

This should be set to True by default based on recent fixes and discussion